docs/CONTRIBUTING.md

```diff
@@ -32,17 +32,20 @@ these guidelines:
    of `git commit`).
  * Keep the number of commits in your request to a minimum. Having multiple
    commits is fine if necessary to separate multiple changes, but make sure
-   that these are logical and value to the commit history.
+   that these are logical and add value to the commit history. For smaller
+   changes, your request may simply be squash merged, making sure to preserve
```

Comment:
You can link to [the helpful guide](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) I used when I was trying to reduce my garbage commit (something like fixing a typo etc).

Aside from this one, I do think this PR is good to go for me. :)

Reply:
Great, thank you very much for reviewing this! I will merge it after I have time to do another full review on it myself, and update the issues I have marked as "help wanted" to link to the Contribution and development guides as starting points.